shogun-lib/src/main/java/de/terrestris/shoguncore/service/security/permission/GroupInstancePermissionService.java

```diff
@@ -53,24 +50,41 @@
         ));
     }
 
-    public PermissionCollection findPermissionCollectionFor(BaseEntity entity, Group group) {
-        Optional<GroupInstancePermission> groupInstancePermission = this.findFor(entity, group);
+    public Optional<GroupInstancePermission> findFor(BaseEntity entity, Group group, User user) {
```

Comment:
Could you add some small code documentation here? Maybe also to the other methods.

Reply:
You're right, documentation is lacking. I'll add it.

Reply:
Added in e134a14.